packages/mcp-server/package.json

```diff
@@ -10,7 +10,9 @@
 	"packageManager": "pnpm@10.15.1",
 	"type": "module",
 	"scripts": {
-		"test": "vitest"
+		"test": "vitest",
+		"generate-summaries": "tsx scripts/generate-summaries.ts",
```

Comment:
Since `node` can now execute typescript files I don't think we need `tsx`.

Reply:
Mmm that's weird...I'm building locally with Node v24.3.0 and it works

Reply:
Let's use `tsx`...it's fine

Reply:
pnpm uses 22 because it's set via `useNodeVersion: 22.19.0` in `pnpm-workspace.yml`

Reply:
You were right, it seems like 22 supports the flag, so I refactored to use it and removed tsx.